args.go

```diff
@@ -105,7 +105,7 @@ func (a *Args) ParseBytes(b []byte) {
 
 // String returns string representation of query args.
 func (a *Args) String() string {
-	return string(a.QueryString())
+	return b2s(a.QueryString())
```

Comment:
I think this is a dangerous change. There could in theory be users that currency use this string and assume it is always safe to use. This change would make this string get modified when the Args get reused. So this change would be backwards incompatible. Therefor I would prefer not to do this.

Reply:
thx for your review